### PR TITLE
Feat: openai and azure-openai realtime api support

### PR DESCRIPTION
**Pull request by VisargD**
Title:

Support for OpenAI realtime API
Related Issues: (optional)

https://github.com/Portkey-AI/gateway/issues/664

### CODE REVIEW

File: src/handlers/realtimeHandler.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { Context } from 'hono';
```
Comment (narengogi) [inline src/handlers/realtimeHandler.ts:1]: we should add exception handling here as well

File: src/services/realtimeLLMEventParser.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
+import { Context } from 'hono';
+
+export class RealTimeLLMEventParser {
```
Comment (narengogi) [inline src/services/realtimeLLMEventParser.ts]: typo in classname

File: src/handlers/websocketUtils.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import { Context } from 'hono';
```
Comment (narengogi) [inline src/handlers/websocketUtils.ts:1]: will delete the unused functions

**Review by narengogi — APPROVED** [review]

File: src/services/realtimeLLMEventParser.ts (hunk 2 above)
Comment (VisargD) [reply src/services/realtimeLLMEventParser.ts]: Done

**Pull request merged**